firebase.json

```diff
@@ -201,6 +201,7 @@
       { "source": "/ui/splash-screen/android-splash-screen", "destination": "/platform-integration/android/splash-screen", "type": 301 },
       { "source": "/ui/widgets/catalog", "destination": "/ui/widgets", "type": 301 },
       { "source": "/ui/widgets/widgetindex", "destination": "/reference/widgets", "type": 301 },
+      { "source": "/wasm", "destination": "/platform-integration/web/wasm", "type": 301 },
```

Comment:
This already exists, right? https://flutter.dev/wasm

Reply:
This `firebase.json` is for `docs.flutter.dev`, the setup for `flutter.dev` is elsewhere. I thought it didn't hurt to add it in case anyone tries `docs.flutter.dev/wasm`.

Reply:
Meh. Let's not. I want to keep the "blast radius" of these redirects small.